### DISCUSSION TITLE
Connected, but not able to arm

### DISCUSSION THREAD

**Discussion by abishur** [taligentx/dscKeybusInterface#298] [dscKeybusInterface Discussion]
I was able to use the unlocker.ino to find out my system (PC1616) still had the installer default code and use that code in the HomeAssistant-MQTT.ino being sure to fill in information as required to connect to wifi and my mqtt broker using home assistant

I added the entries to my .yaml file and can see my zones triggering as I open and close doors... but now what?  I hit the "arm" button and nothing happens.  I suspect that my system was installed when the house was built, but was never configured.  Do I need to go through some kind of setup prior to being able to use it?

Also, on the example for home assistant it shows everything as being part of partition 1, but I have nothing there, how do you get your zones to show up in partition 1?

**Comment by rjboonstra** [comment]
Hello... I am using an ESP32 and the "DSC-HomeSpan" program... and have a similar issue.  I can monitor the status of my PC1832 DSC security system and it works very well.  When I try to arm, though, nothing happens.  I used a digital analyzer (Saleae Software and an economical analyzer), and I can see the clock and the data channel transmitting as they should.  The third channel is the virtual keypad... but it does not seem to be enough data nor does it work.
<img width="1253" alt="Screen Shot 2022-07-30 at 9 14 06 PM" src="https://user-images.githubusercontent.com/40444165/182008271-5663e794-cf7d-4e64-b075-a75647eec888.png">
